src/nvim/drawscreen.c

```diff
@@ -699,42 +699,42 @@ static void win_redr_border(win_T *wp)
   if (adj[0]) {
     grid_puts_line_start(grid, 0);
     if (adj[3]) {
-      grid_put_schar(grid, 0, 0, chars[0], attrs[0]);
+      grid_put_schar(grid, 0, 0, (char_u *)chars[0], attrs[0]);
```

Comment:
`grid_put_schar` should be changed to take a `char *` (along with the other schar functions you changed)